clap_builder/texts/en.yaml

```diff
@@ -0,0 +1,10 @@
+clap.usage.header-text: Usage
```

Comment:
These are the hard-coded default texts that I extracted. You'll note that there are many more texts than these -- I just extracted enough to demonstrate using the example that I included in this PR.